src/libraries/System.Net.HttpListener/src/System/Net/Managed/HttpResponseStream.Managed.cs

```diff
@@ -162,7 +162,7 @@ internal void InternalWrite(byte[] buffer, int offset, int count)
 
         private async Task InternalWriteIgnoreErrorsAsync(byte[] buffer, int offset, int count)
         {
-            try { await _stream.WriteAsync(buffer, offset, count).ConfigureAwait(false); }
+            try { await _stream.WriteAsync(buffer.AsMemory(offset, count)).ConfigureAwait(false); }
```

Comment:
This isn't from the cancellation analyzer but rather from the other analyzer you wrote, correct?

Reply:
Correct, it's a case I didn't find before. I can submit it in a separate PR if needed.

Reply:
Nah, it's fine here.